src/Elmish.WPF/Binding.fs

```diff
@@ -188,7 +188,9 @@ module internal BindingData =
         GetState = unbox >> d.GetState
         GetBindings = d.GetBindings
         ToMsg = d.ToMsg >> box
-        GetWindow = d.GetWindow
+        GetWindow =
+          let getWin = d.GetWindow  // No idea why this is needed
```

Comment:
I'm particularly puzzled by why this temporary copy is necessary. Won't compile otherwise.

Reply:
It is because you shadowed `d`.  Don't worry if you think that was a particularly dumb mistake.  It still took me ~10 minutes to figure that out :(

Reply:
Thanks! 🤣 Totally did not see that.